docs/source/media_upload/index.rst

```diff
@@ -0,0 +1,66 @@
+.. _Media Upload:
+
+Media Upload
+============
+
+.. image:: ./images/media_upload.png
+
+Piccolo Admin has the option of uploading media files to local static folder 
+(local storage). Multi file upload is enabled per record for all columns 
+specified in ``media_columns``. 
+
+
+Usage
+-----
+
+Piccolo admin uses the Piccolo ORM `Array <https://piccolo-orm.readthedocs.io/en/latest/piccolo/schema/column_types.html#array>`_ 
```

Comment:
I'm not 100% sure about it having to be an `Array` column. I can imagine situations where someone wants an array, but I can also imagine situations where someone wants to use a `Varchar`, as they only want a single image to be uploaded.

Reply:
The input type file is set to upload only one file, so the user can easily upload only one image. For uploading multiple files `Array` column is ideal (we already have a `ArrayWidget` for adding and removing fields which is very useful), so this solves the uploading of multiple or single file in one solution.

Reply:
Yeah, in some situations though you want to make sure the user can only upload one file.

Imagine you're building a user profile table, and you have a field called 'profile_image'. If there are multiple images stored, it will be confusing - which is the genuine one?

Reply:
I made 2 videos to see what I mean.
Single file upload

https://user-images.githubusercontent.com/30960668/180474565-79feda60-3c0d-4bc0-9ebe-6a6e3428f882.mp4

Multiple files upload


https://user-images.githubusercontent.com/30960668/180474867-9a63121c-bb8e-41b2-95df-0a3d27f30994.mp4

But of course you can do it however you want, but I think this is a simple and efficient solution because we actually upload only one file each time.